container_checkpoint_test.go

```diff
@@ -408,3 +415,126 @@ func TestCheckpointLeaveRunning(t *testing.T) {
 
 	<-statusC
 }
+
+func withCheckpointOpts(rt, path string) CheckpointTaskOpts {
+	return func(r *CheckpointTaskInfo) error {
+		switch rt {
+		case v1runtime:
+			r.Options = &runctypes.CheckpointOptions{
+				ImagePath: path,
+			}
+		case v2runtime:
+			r.Options = &options.CheckpointOptions{
+				ImagePath: path,
+			}
+		}
+		return nil
+	}
+}
+
+func withRestoreOpts(rt, path string) NewTaskOpts {
```

Comment:
This would also be nice as a package opt for other clients.  Maybe something like `WithRestoreImagePath`.